platform/android/os_android.h

```diff
@@ -90,6 +90,8 @@ class OS_Android : public OS_Unix {
 
 	virtual Error open_dynamic_library(const String p_path, void *&p_library_handle, bool p_also_set_library_path = false);
 
+	virtual Rect2 get_window_safe_area() const;
```

Comment:
I missed that before, but on 4.0, this method has been replaced with `DisplayServer::screen_get_usable_rect`. You can check how it's done for ios as an example.

Reply:
ah, I've renamed everything to `screen_get_usable_rect` and moved it to DisplayServer.
I still have the 3.2 branch if that helps with back porting :man_shrugging:

Reply:
Yeah, it's worth making a separate PR for the 3.2 branch in this case.